utils/src/wizard/InputRevalidationStep.ts

```diff
@@ -0,0 +1,30 @@
+/*---------------------------------------------------------------------------------------------
+ *  Copyright (c) Microsoft Corporation. All rights reserved.
+ *  Licensed under the MIT License. See License.txt in the project root for license information.
+ *--------------------------------------------------------------------------------------------*/
+
+import { AzureWizardPromptStep, IActionContext } from "../..";
+import { localize } from "../localize";
+
+export class InputRevalidationStep<T extends IActionContext> extends AzureWizardPromptStep<T> {
+    constructor(private readonly _key: string, private readonly _isPassword?: boolean) {
```

Comment:
nit: remove underscores

Reply:
I thought we were using underscores to represent private properties but excluding them on private methods.  What exactly is the standard convention again so I don't keep messing this up 😅

Reply:
I thought it was for private properties that are exposed via a public getter.

Like if the property represents an internal value for something that is also exposed publicly, then add an underscore to avoid a naming conflict.